post_installation_configuration/storage-configuration.adoc

```diff
@@ -70,8 +70,8 @@ include::modules/available-persistent-storage-options.adoc[leveloffset=+1]
 include::modules/recommended-configurable-storage-technology.adoc[leveloffset=+1]
 
 [id="post-install-deploy-OCS"]
-== Deploy Red Hat OpenShift Container Storage
+== Deploy Red Hat OpenShift Data Foundation
 // This section is sourced from storage/persistent_storage/persistent-storage-ocs.adoc
-Red Hat OpenShift Container Storage is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, Red Hat OpenShift Container Storage is completely integrated with {product-title} for deployment, management, and monitoring.
+{rh-storage-first} is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, {rh-storage-first} is completely integrated with {product-title} for deployment, management, and monitoring.
 
-include::modules/deploy-red-hat-openshift-container-storage.adoc[leveloffset=+1]
+include::modules/deploy-red-hat-openshift-data-foundation.adoc[leveloffset=+1]
```

Comment:
You didn't actually change the filename itself, so this is probably why the build is failing.

Reply:
Yeah, and I would suggest not changing the names of the files, because it will change the URL for these pages and cause 404s or the needs for redirects. Changing the titles in the files should be sufficient.

Reply:
+1 to not changing the filename. Even though it's a mismatch, this is fairly common in our docs and not really an impact to the customer.

Reply:
thanks for the help!

Reply:
@bobfuru @bergerhoffer I fixed this issue, but the build is still failing. I can't quite understand the build error log--any ideas why this is still failing?